src/python/WMCore/Lexicon.py

```diff
@@ -656,3 +667,24 @@ def activity(candidate):
     if candidate in dashboardActivities:
         return True
     raise AssertionError("Invalid dashboard activity: %s should 'test'" % candidate)
+
+
+def getStringsBetween(start, end, source):
```

Comment:
Are we planning to use this method or it's just a leftover from your initial work?

Reply:
I thought it might be useful in some case to extract some thing between tag. I am not currently using is though. If you prefer I can remove it.

Reply:
I'm not a big fan of writing code that might one day be used (I guess usually they are not). Everyone has a different taste though, and it's small. we can give it a shot :)